Tests/Unit/LoRaTools/RegionTestBaseRxpk.cs

```diff
@@ -125,5 +125,18 @@ protected void TestTryGetJoinChannelIndexRxpk(string dr, double freq, int expect
             Assert.False(Region.TryGetJoinChannelIndex(rxpk[0], out var channelIndex));
             Assert.Equal(expectedIndex, channelIndex);
         }
+
+        protected void TestIsDRValueWithinAcceptableValues(string dataRate, bool upstream, bool isValid)
```

Comment:
Adding coverage for untested methods `IsCurrentUpstreamDRValueWithinAcceptableValue` and ` IsCurrentDownstreamDRValueWithinAcceptableValue`.